app/src/app/store/packets/packets.ts

```diff
@@ -14,7 +14,7 @@ export const packetsSlice = createSlice({
     extraReducers: (builder) => {
         builder
             .addCase(actions.fetchPackets.fulfilled, (state, action) => {
-                state.packets = action.payload.sort((a, b) => a.name.localeCompare(b.name));
+                state.packets = action.payload;
```

Comment:
Sorry, I don't really understand this change. I wasn't suggesting that you change the initial ordering from sorted by name, that seems fine. I was just saying that when the user then wants to sort by another column (clicks on 'Status'), it should sort ascending since that's the first Status sort. 

It should probably toggle the ascending flag on sort, unless a new column from the previous sort has been selected, in which case it should always revert to ascending = true.

I might not have explained this very well, happy to chat about it!